core-web/libs/ui/src/lib/directives/dot-select-item/dot-select-item.directive.ts

```diff
@@ -0,0 +1,26 @@
+import { Directive, HostListener, Optional, Self } from '@angular/core';
+
+import { AutoComplete } from 'primeng/autocomplete';
+
+@Directive({
+    selector: 'p-autoComplete[dotSelectItem]',
```

Comment:
Will this be a global behavior or just for this case? If it is global, I could leave `p-autocomplete`  because I could forget to add the `dotSelectItem`

Reply:
I don't know if will be for 100% of the cases... I guess we can see if we have in a 3rd case we will revisit.